src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -164,6 +170,11 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
                 context = application.CreateContext(invocationFeatures);
                 invocationFeatures.Set<IFunctionBindingsFeature>(new GrpcFunctionBindingsFeature(context, request, outputBindingsInfoProvider));
 
+                if (functionInputConversionFeatureProvider.TryCreate(typeof(DefaultInputConversionFeature), out var conversion))
```

Comment:
I don't think this should be happening here -- we don't want to create one of these for every invocation, do we? Going to think through this a bit...